test/snippet/core/type_list/list_traits_at.cpp

```diff
@@ -0,0 +1,11 @@
+#include <seqan3/core/type_list/traits.hpp>
+
+int main()
+{
+    using t = seqan3::type_list<int, float, bool, double>;
```

Comment:
Please call it `list_t` everywhere. I would like to avoid one letter variables in the snippets